mentorApp/lib/main.dart

```diff
@@ -1,5 +1,7 @@
 import 'package:flutter/material.dart';
+import 'SignUp/FTU.dart';
 import 'AppController.dart';
+import 'Profile/UserProfile.dart';
```

Comment:
Not a big deal, but is this import being used anywhere in the file?

Reply:
Oh sorry, I was using it before when I was trying to route properly but it's not needed now.